docs/medical-api/api-reference/document/download-url-bulk.mdx

```diff
@@ -60,8 +60,8 @@ const metadata = {
   stringKeyValue: "pairsHere",
 };
 
-const status = await metriportClient.startBulkGetDocumentUrl(
-  "<your-patient-id>",
+const status = await api.startBulkGetDocumentUrl(
```

Comment:
You left `metriportClient` above though?

Reply:
Good catch -- fixed and aligned with patient / facility.